test/op-worker-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+
+$SCRIPT
+
+EXPECTED_FLAVOR=observability-pipelines-worker
+INSTALLED_VERSION=
+RESULT=0
+EXPECTED_MAJOR_VERSION=1
+EXPECTED_MINOR_VERSION="${EXPECTED_MINOR_VERSION:-${DD_OP_WORKER_MINOR_VERSION}}"
+
+if command -v dpkg > /dev/null; then
+  INSTALLED_VERSION=$(dpkg-query -W ${EXPECTED_FLAVOR} | cut -f2 | cut -d: -f2)
+else
+  INSTALLED_VERSION=$(rpm -q --queryformat "%{version}" "${EXPECTED_FLAVOR}")
+fi
```

Comment:
The `localtest.sh` also tests the installed package for changes, however since we're currently updating the `bootstrap.yaml` in place - this would always fail.